helix-view/src/document.rs

```diff
@@ -344,6 +349,7 @@ impl Document {
             history: Cell::new(History::default()),
             savepoint: None,
             last_saved_revision: 0,
+            modified_since_accessed: Cell::new(false),
```

Comment:
Why does this need to be a `Cell`? `std::mem::take()` should work just fine

Reply:
Thanks I forgot about `mem::take`.